convert/src/proto.rs

```diff
@@ -0,0 +1,13 @@
+/// Convert a SystemTime to a protobuf timestamp. Not sure about the correctness of nanoseconds calculation
+pub fn systime_to_timestamp(t: std::time::SystemTime) -> prost_types::Timestamp {
```

Comment:
https://docs.rs/prost-types/0.8.0/prost_types/struct.Timestamp.html
Timestamp already implements From<SysTime> ...
consider using Instant instead of SysTime, even if we do, there's still a cleaner example of conversion in the prost_types docs above.
(kind of the same for the opposite conversion)

Reply:
Won't use Instant atm, will use Prost builtin trait instead